src/index.ts

```diff
@@ -1,15 +1,31 @@
-import { Config, Client } from "./shared-types.js";
-import { HttpDriver } from "./http/http-driver.js";
-import { SqliteDriver } from "./sqlite-driver.js";
+import type { Config, Client } from "./api.js";
+import { LibsqlError } from "./api.js";
+import { expandConfig } from "./config.js";
+import { createClient as createHranaClient } from "./hrana.js";
+import { createClient as createHttpClient } from "./http.js";
+import { createClient as createSqlite3Client } from "./sqlite3.js";
+
+export * from "./api.js";
 
 export function createClient(config: Config): Client {
```

Comment:
Just checking to make sure that a client can still `import { createClient } from "@libsql/client/http"` to get a version of the client that bypasses sqlite3 entirely for use in environments that do not fully emulate node.

Reply:
Yes, that was the goal, though I haven't checked it yet. We will probably also need an import such as `"@libsql/client/web"` that uses only HTTP and Hrana (the Web APIs `fetch` and `WebSocket`); most environments that support `fetch()` also support `WebSocket` (such as Cloudflare Workers or Deno), so users should be able to use both protocols, not just HTTP.

Reply:
It sounds like @libsql/client/web would replace @libsql/client/http entirely.

Reply:
Aaaand we've come full circle in a private chat.  What we'll do next is have @libsql/client/web which imports both @libsql/client/http and @libsql/client/hrana and decides which of them to use based on the URL scheme.